subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -54,7 +54,7 @@
         .changedTo(68, "5.0-milestone-1")
         .changedTo(69, "5.0-rc-1")
         .changedTo(71, "5.3-rc-1")
-        .changedTo(77, "6.0-rc-1")
+        .changedTo(78_456_123, "6.0-rc-1")
```

Comment:
Is this supposed to be some crazy number?

Reply:
ah, I see Jendrik's comment now.

Reply:
Yeah, will become 78 then I merge 😉